src/app/api/v1/hibp/notify/route.ts

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest, NextResponse } from "next/server";
+import { bearerToken } from "../../../utils/auth";
+
+import { PubSub } from "@google-cloud/pubsub";
+
+const projectId = "rhelmer-monitor-local-dev";
```

Comment:
Is this for everybody, or just your local env?

Reply:
Oops, I meant to change this to use the env vars, not just the subscriber. Thanks!